src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery youCanWinMillionDollars = new Lottery();
+        int i = 0;
+        while (i < 3) {
+            System.out.println(youCanWinMillionDollars.getRandomBall());
+            i++;
+        }
```

Comment:
replace it with for loop
```suggestion
        Lottery youCanWinMillionDollars = new Lottery();
        int i = 0;
        while (i < 3) {
            System.out.println(youCanWinMillionDollars.getRandomBall());
            i++;
        }
```